clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -727,6 +730,16 @@ private void checkGroupRemoteAssignor() {
         }
     }
 
+    private void checkPartitionAssigmentStrategy() {
+        List<String> assignmentStrategies = getList(PARTITION_ASSIGNMENT_STRATEGY_CONFIG);
+        if (getString(GROUP_PROTOCOL_CONFIG).equalsIgnoreCase(GroupProtocol.CONSUMER.name())) {
+            List<Class<? extends AbstractPartitionAssignor>> defaultValue = Arrays.asList(RangeAssignor.class, CooperativeStickyAssignor.class);
+            if (!new HashSet<>(assignmentStrategies).containsAll(defaultValue) || assignmentStrategies.size() != defaultValue.size()) {
```

Comment:
maybe we should accept the config equal to the default. for example, the config having `org.apache.kafka.clients.consumer.RangeAssignor` and `org.apache.kafka.clients.consumer.CooperativeStickyAssignor` should work